### PR TITLE
Refactor tests

### PR DESCRIPTION
**Pull request by jasonrhansen**

### CODE REVIEW

File: src/bin/flamegraph.rs [hunk 1]
```diff
@@ -61,6 +61,17 @@ struct Opt {
     /// Verbose logging mode (-v, -vv, -vvv)
     #[structopt(short = "v", long = "verbose", parse(from_occurrences))]
     verbose: usize,
+
+    /// Pretty print XML with new lines and indentation.
```
Comment (jonhoo) [inline src/bin/flamegraph.rs]:
```suggestion
    /// Pretty print XML with newlines and indentation.
```

File: tests/data/collapse-perf/inline-counter.txt [hunk 2]
```diff
@@ -0,0 +1,243 @@
+inline-counter 26037  8801.193547:     250000 cpu-clock:uhH:
```
Comment (jonhoo) [inline tests/data/collapse-perf/inline-counter.txt]: I'm not entirely sure why this file seems to only have been added now?

**Comment by jonhoo** [comment]
LGTM subject to those two minor comments.

File: tests/data/collapse-perf/inline-counter.txt (hunk 2 above)
Comment (jasonrhansen) [reply tests/data/collapse-perf/inline-counter.txt]: I moved the file and had to update the path to the binary everywhere it was referenced.

Comment (jonhoo) [reply tests/data/collapse-perf/inline-counter.txt]: Ah, I just missed its removal is all!

**Pull request merged**